src/org/rascalmpl/library/vis/Charts.rsc

```diff
@@ -344,6 +344,7 @@ data ChartType
 data ChartOptions  
     = chartOptions(
         bool responsive=true,
+        bool animations=false,
```

Comment:
pretty sure this meant to go in a different PR?

Reply:
yes this is an accident. changing it back!